draft-yasskin-http-origin-signed-responses.md

```diff
@@ -481,97 +487,103 @@ following CDDL ({{!I-D.ietf-cbor-cddl}}):
 validity = {
   ? signatures: [ + bytes ]
   ? update: {
-    url: text,
     ? size: uint,
   }
 ]
 ~~~
 
-The elements of the `signatures` array are header field values meant to replace
-the signatures within the `Signature` header field pointing to this validity
-data. If the signed exchange contains a bug severe enough that clients need to
-stop using the content, the `signatures` array MUST NOT be present.
+The elements of the `signatures` array are parameterised labels (Section 4.4 of
+{{!I-D.ietf-httpbis-header-structure}}) meant to replace the signatures within
+the `Signature` header field pointing to this validity data. If the signed
+exchange contains a bug severe enough that clients need to stop using the
+content, the `signatures` array MUST NOT be present.
 
-The `update` map gives a location to update the entire signed exchange and an
-estimate of the size of the resource at that URL. If the signed exchange is
-currently the most recent version, the `update` SHOULD NOT be present.
+If the the `update` map is present, that indicates that a new version of the
+signed exchange is available at its effective request URI (Section 5.5 of
+{{!RFC7230}}) and can give an estimate of the size of the updated exchange
+(`update.size`). If the signed exchange is currently the most recent version,
+the `update` SHOULD NOT be present.
 
 If both the `signatures` and `update` fields are present, clients can use the
 estimated size to decide whether to update the whole resource or just its
 signatures.
 
 ### Examples ### {#examples-updating-validity}
 
-For example, if a signed exchange has the following `Signature` header field (written as multiple fields for convenience):
+For example, if a signed exchange whose URL is `https://example.com/resource`
+has the following `Signature` header field (with line breaks for ease of
+reading):
 
 ~~~http
 Signature: sig1;
   sig=*MEUCIQDXlI2gN3RNBlgFiuRNFpZXcDIaUpX6HIEwcZEc0cZYLAIga9DsVOMM+g5YpwEBdGW3sS+bvnmAJJiSMwhuBdqp5UY;
   integrity="mi";
-  validityUrl="https://example.com/resource.validity";
+  validityUrl="https://example.com/resource.validity.1511128380";
   certUrl="https://example.com/certs";
   certSha256=*W7uB969dFW3Mb5ZefPS9Tq5ZbH5iSmOILpjv2qEArmI;
-  date=1511128380; expires=1511560380
-Signature: sig2;
+  date=1511128380; expires=1511560380,
+  sig2;
   sig=*MEQCIGjZRqTRf9iKNkGFyzRMTFgwf/BrY2ZNIP/dykhUV0aYAiBTXg+8wujoT4n/W+cNgb7pGqQvIUGYZ8u8HZJ5YH26Qg;
   integrity="mi";
-  validityUrl="https://example.com/resource.validity";
+  validityUrl="https://example.com/resource.validity.1511128380";
   certUrl="https://example.com/certs";
   certSha256=*kQAA8u33cZRTy7RHMO4+dv57baZL48SYA2PqmYvPPbg;
-  date=1511301183; expires=1511905983
-Signature: sig3;
+  date=1511301183; expires=1511905983,
```

Comment:
As a further comment re: examples, it's unclear what the implications of the overlap are between the expires on sig1 and the date on sig2. That is, it's not clear the use case for the overlapping set of signatures with the same validityURL, versus discrete (non-overlapping)

This is not specific to this PR, but was within the set of concerns with the examples being relatively opaque. It may be that an 'explainer' type document is better suited to capture this nuance, but as an example, it's a bit confusing :)